deps/llvm.mk

```diff
@@ -431,6 +431,7 @@ endif
 $(eval $(call LLVM_PATCH,llvm-D51842-win64-byval-cc))
 $(eval $(call LLVM_PATCH,llvm-D57118-powerpc))
 $(eval $(call LLVM_PATCH,llvm-r355582-avxminmax)) # remove for 8.0
+$(eval $(call LLVM_PATCH,llvm-AMDGPU-addrspaces)) # AMDGPU
```

Comment:
misses the WASM patch

Reply:
I was wondering about this; since the patch is generated against 7.0, should I just put it in the 7.0 section? The same question goes for the AMDGPU patch as well, which we'd want working on any version of LLVM we plan to support.

Reply:
Most often than not patches are applicable accross versions and should be apply to all that don't have them upstreamed

Reply:
In that case, I'll just make multiple patches if I find one of them doesn't apply on a different version. I know I tried the WASM patch on a different version last night and it failed to patch.